api/envoy/config/trace/v2/trace.proto

```diff
@@ -136,6 +136,13 @@ message OpenCensusConfig {
   // The URL to Zipkin, e.g. "http://127.0.0.1:9411/api/v2/spans"
   string zipkin_url = 6;
 
+  // Enables the OpenCensus Agent exporter if set to true. The address must also
+  // be set.
+  bool ocagent_exporter_enabled = 11;
+
+  // The address of the OpenCensus Agent, if its exporter is enabled.
+  string ocagent_address = 12;
```

Comment:
What kind of address is this? URL? hostname? IP?

Reply:
Updated comment. (it's in gRPC format, just like the SD address)